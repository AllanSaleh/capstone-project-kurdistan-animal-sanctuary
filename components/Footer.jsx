import React from "react";
import { useTranslation } from "next-i18next";
import { SocialIcon } from "react-social-icons";

export default function footer() {
  const { t } = useTranslation("footer");
  return (
    // eslint-disable-next-line react/react-in-jsx-scope
    <footer className="w-full h-96 pt-10 bg-secondary text-2xl bg-paw  bg-no-repeat  bg-cover font-Quicksand">
      <div className=" md:block-inline mx-5 justify-center mx-8 md:mx-0 md:pl-56 md:py-5">
        <div className=" hidden w-20 md:h-40 md:w-40 md:inline-block relative ">
          <img
            src="/logo.png"
            alt="logo"
            className="justify-self-end m-auto w-20 h-20 md:w-36 md:h-36  "
          />
        </div>
        <div className=" w-11/12 md:w-96 mb-5 md:mb-0 md:pt-5 md:pl-24 inline-block text-lg md:text-2xl ">
          <h1 className=" text-primary font-bold font-Quicksand">
            {t("footer.aboutus")}
          </h1>
          <p className="  text-primary font-Quicksand ">
            {t("footer.aboutUsDescrption")}
          </p>
        </div>
        <div className=" md:pt-5 md:pl-36 inline-block text-lg md:text-2xl">
          <h1 className=" text-primary font-bold font-Quicksand">
            {t("footer.getInTouch")}
          </h1>
          <p className=" text-primary font-Quicksand ">
            info@kas.com <br />
            964 123 456 7890 <br />
            KRG, Erbil, Iraq
            <br />
          </p>
          <div>
            <ul className="flex flex-row mt-4">
              <li>
                <SocialIcon
                  url="twitter.com"
                  network="twitter"
                  bgColor="#056676"
                />
              </li>
              <li className="pl-1">
                <SocialIcon
                  url="https://www.instagram.com"
                  network="instagram"
                  bgColor="#056676"
                />
              </li>
              <li className="pl-1">
                <SocialIcon
                  url="https://www.facebook.com"
                  network="facebook"
                  bgColor="#056676"
                />
              </li>
            </ul>
          </div>
        </div>
      </div>
    </footer>
  );
}
