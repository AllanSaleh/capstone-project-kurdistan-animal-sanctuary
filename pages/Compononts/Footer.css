.col {
  float: left;
  width: 33.33%;
  padding: 5px;
}
.row::after {
  content: "";
  display: table;
  clear: both;
}
