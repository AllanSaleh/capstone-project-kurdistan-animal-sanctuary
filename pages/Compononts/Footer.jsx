import React from "react";

export default function footer() {
  return (
    // eslint-disable-next-line react/react-in-jsx-scope
    <div className="bg-secondary">
      <div class="row">
        <img src="logo.png" />
      </div>
      <div class="col">
        <img src="Paw.png" />
      </div>
      <h1 className="text-left text-primary text-3xl font-bold font-Quicksand ">
        Contact Us
      </h1>
      <p className="text-left p-6 text-primary px-72 font-Quicksand ">
        Kurdistan Animal Sanctacy (KAS) <br />
        is NGO that saves <br />
        animals and looks for a loving
        <br />
        parent to adopt them
      </p>
      <h1 className="text-left text-primary text-3xl font-bold font-Quicksand ">
        Get in touch
      </h1>
      <p className="text-left p-6 text-primary px-72 font-Quicksand ">
        info@kas.com <br />
        964 123 456 7890 <br />
        KRG, Erbil, Iraq <br />
      </p>
    </div>
  );
}
