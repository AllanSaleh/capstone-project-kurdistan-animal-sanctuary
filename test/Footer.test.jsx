import React from "react";
import renderer from "react-test-renderer";
import { test, expect } from "@jest/globals";
import Footer from "../components/Footer";

test("proparly renders the Footer component", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
