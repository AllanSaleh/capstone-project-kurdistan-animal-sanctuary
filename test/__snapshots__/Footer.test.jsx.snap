// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proparly renders the Footer component 1`] = `
<footer
  className="w-full h-96 pt-10 bg-secondary text-2xl bg-paw  bg-no-repeat  bg-cover font-Quicksand"
>
  <div
    className=" md:block-inline mx-5 justify-center mx-8 md:mx-0 md:pl-56 md:py-5"
  >
    <div
      className=" hidden w-20 md:h-40 md:w-40 md:inline-block relative "
    >
      <img
        alt="logo"
        className="justify-self-end m-auto w-20 h-20 md:w-36 md:h-36  "
        src="/logo.png"
      />
    </div>
    <div
      className=" mb-5 md:mb-0 md:pt-5 md:pl-24 inline-block text-lg md:text-2xl w-96"
    >
      <h1
        className=" text-primary font-bold font-Quicksand"
      >
        footer.aboutus
      </h1>
      <p
        className="  text-primary font-Quicksand "
      >
        footer.aboutUsDescrption
      </p>
    </div>
    <div
      className=" md:pt-5 md:pl-36 inline-block text-lg md:text-2xl"
    >
      <h1
        className=" text-primary font-bold font-Quicksand"
      >
        footer.getInTouch
      </h1>
      <p
        className=" text-primary font-Quicksand "
      >
        info@kas.com 
        <br />
        964 123 456 7890 
        <br />
        KRG, Erbil, Iraq
        <br />
      </p>
      <div>
        <ul
          className="flex flex-row mt-4"
        >
          <li>
            <a
              aria-label="twitter"
              className="social-icon"
              href="twitter.com"
              style={
                Object {
                  "display": "inline-block",
                  "height": "50px",
                  "overflow": "hidden",
                  "position": "relative",
                  "verticalAlign": "middle",
                  "width": "50px",
                }
              }
            >
              <div
                className="social-container"
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "position": "absolute",
                    "top": 0,
                    "width": "100%",
                  }
                }
              >
                <svg
                  className="social-svg"
                  style={
                    Object {
                      "borderRadius": "50%",
                      "fillRule": "evenodd",
                      "height": "100%",
                      "left": 0,
                      "position": "absolute",
                      "top": 0,
                      "width": "100%",
                    }
                  }
                  viewBox="0 0 64 64"
                >
                  <g
                    className="social-svg-background"
                    style={
                      Object {
                        "MozTransition": "fill 170ms ease-in-out",
                        "OTransition": "fill 170ms ease-in-out",
                        "WebkitTransition": "fill 170ms ease-in-out",
                        "fill": "transparent",
                        "msTransition": "fill 170ms ease-in-out",
                        "transition": "fill 170ms ease-in-out",
                      }
                    }
                  >
                    <circle
                      cx="32"
                      cy="32"
                      r="31"
                    />
                  </g>
                  <g
                    className="social-svg-icon"
                    style={
                      Object {
                        "MozTransition": "fill 170ms ease-in-out",
                        "OTransition": "fill 170ms ease-in-out",
                        "WebkitTransition": "fill 170ms ease-in-out",
                        "fill": "transparent",
                        "msTransition": "fill 170ms ease-in-out",
                        "transition": "fill 170ms ease-in-out",
                      }
                    }
                  >
                    <path
                      d="M48,22.1c-1.2,0.5-2.4,0.9-3.8,1c1.4-0.8,2.4-2.1,2.9-3.6c-1.3,0.8-2.7,1.3-4.2,1.6 C41.7,19.8,40,19,38.2,19c-3.6,0-6.6,2.9-6.6,6.6c0,0.5,0.1,1,0.2,1.5c-5.5-0.3-10.3-2.9-13.5-6.9c-0.6,1-0.9,2.1-0.9,3.3 c0,2.3,1.2,4.3,2.9,5.5c-1.1,0-2.1-0.3-3-0.8c0,0,0,0.1,0,0.1c0,3.2,2.3,5.8,5.3,6.4c-0.6,0.1-1.1,0.2-1.7,0.2c-0.4,0-0.8,0-1.2-0.1 c0.8,2.6,3.3,4.5,6.1,4.6c-2.2,1.8-5.1,2.8-8.2,2.8c-0.5,0-1.1,0-1.6-0.1c2.9,1.9,6.4,2.9,10.1,2.9c12.1,0,18.7-10,18.7-18.7 c0-0.3,0-0.6,0-0.8C46,24.5,47.1,23.4,48,22.1z"
                    />
                  </g>
                  <g
                    className="social-svg-mask"
                    style={
                      Object {
                        "MozTransition": "fill 170ms ease-in-out",
                        "OTransition": "fill 170ms ease-in-out",
                        "WebkitTransition": "fill 170ms ease-in-out",
                        "fill": "#056676",
                        "msTransition": "fill 170ms ease-in-out",
                        "transition": "fill 170ms ease-in-out",
                      }
                    }
                  >
                    <path
                      d="M0,0v64h64V0H0z M44.7,25.5c0,0.3,0,0.6,0,0.8C44.7,35,38.1,45,26.1,45c-3.7,0-7.2-1.1-10.1-2.9 c0.5,0.1,1,0.1,1.6,0.1c3.1,0,5.9-1,8.2-2.8c-2.9-0.1-5.3-2-6.1-4.6c0.4,0.1,0.8,0.1,1.2,0.1c0.6,0,1.2-0.1,1.7-0.2 c-3-0.6-5.3-3.3-5.3-6.4c0,0,0-0.1,0-0.1c0.9,0.5,1.9,0.8,3,0.8c-1.8-1.2-2.9-3.2-2.9-5.5c0-1.2,0.3-2.3,0.9-3.3 c3.2,4,8.1,6.6,13.5,6.9c-0.1-0.5-0.2-1-0.2-1.5c0-3.6,2.9-6.6,6.6-6.6c1.9,0,3.6,0.8,4.8,2.1c1.5-0.3,2.9-0.8,4.2-1.6 c-0.5,1.5-1.5,2.8-2.9,3.6c1.3-0.2,2.6-0.5,3.8-1C47.1,23.4,46,24.5,44.7,25.5z"
                    />
                  </g>
                </svg>
              </div>
            </a>
          </li>
          <li
            className="pl-1"
          >
            <a
              aria-label="instagram"
              className="social-icon"
              href="https://www.instagram.com"
              style={
                Object {
                  "display": "inline-block",
                  "height": "50px",
                  "overflow": "hidden",
                  "position": "relative",
                  "verticalAlign": "middle",
                  "width": "50px",
                }
              }
            >
              <div
                className="social-container"
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "position": "absolute",
                    "top": 0,
                    "width": "100%",
                  }
                }
              >
                <svg
                  className="social-svg"
                  style={
                    Object {
                      "borderRadius": "50%",
                      "fillRule": "evenodd",
                      "height": "100%",
                      "left": 0,
                      "position": "absolute",
                      "top": 0,
                      "width": "100%",
                    }
                  }
                  viewBox="0 0 64 64"
                >
                  <g
                    className="social-svg-background"
                    style={
                      Object {
                        "MozTransition": "fill 170ms ease-in-out",
                        "OTransition": "fill 170ms ease-in-out",
                        "WebkitTransition": "fill 170ms ease-in-out",
                        "fill": "transparent",
                        "msTransition": "fill 170ms ease-in-out",
                        "transition": "fill 170ms ease-in-out",
                      }
                    }
                  >
                    <circle
                      cx="32"
                      cy="32"
                      r="31"
                    />
                  </g>
                  <g
                    className="social-svg-icon"
                    style={
                      Object {
                        "MozTransition": "fill 170ms ease-in-out",
                        "OTransition": "fill 170ms ease-in-out",
                        "WebkitTransition": "fill 170ms ease-in-out",
                        "fill": "transparent",
                        "msTransition": "fill 170ms ease-in-out",
                        "transition": "fill 170ms ease-in-out",
                      }
                    }
                  >
                    <path
                      d="M 39.88,25.89 C 40.86,25.89 41.65,25.10 41.65,24.12 41.65,23.14 40.86,22.35 39.88,22.35 38.90,22.35 38.11,23.14 38.11,24.12 38.11,25.10 38.90,25.89 39.88,25.89 Z M 32.00,24.42 C 27.82,24.42 24.42,27.81 24.42,32.00 24.42,36.19 27.82,39.58 32.00,39.58 36.18,39.58 39.58,36.18 39.58,32.00 39.58,27.82 36.18,24.42 32.00,24.42 Z M 32.00,36.92 C 29.28,36.92 27.08,34.72 27.08,32.00 27.08,29.28 29.28,27.08 32.00,27.08 34.72,27.08 36.92,29.28 36.92,32.00 36.92,34.72 34.72,36.92 32.00,36.92 Z M 32.00,19.90 C 35.94,19.90 36.41,19.92 37.96,19.99 39.41,20.05 40.19,20.29 40.71,20.50 41.40,20.77 41.89,21.08 42.41,21.60 42.92,22.12 43.24,22.61 43.51,23.30 43.71,23.82 43.95,24.60 44.02,26.04 44.09,27.60 44.11,28.06 44.11,32.01 44.11,35.95 44.09,36.41 44.02,37.97 43.95,39.41 43.71,40.19 43.51,40.71 43.24,41.40 42.92,41.90 42.41,42.41 41.89,42.93 41.40,43.25 40.71,43.51 40.19,43.71 39.41,43.96 37.96,44.02 36.41,44.09 35.94,44.11 32.00,44.11 28.06,44.11 27.59,44.09 26.04,44.02 24.59,43.96 23.81,43.72 23.29,43.51 22.60,43.24 22.11,42.93 21.59,42.41 21.08,41.90 20.76,41.40 20.49,40.71 20.29,40.19 20.05,39.41 19.98,37.97 19.91,36.41 19.89,35.95 19.89,32.01 19.89,28.06 19.91,27.60 19.98,26.04 20.05,24.60 20.29,23.82 20.49,23.30 20.76,22.61 21.08,22.12 21.59,21.60 22.11,21.08 22.60,20.76 23.29,20.50 23.81,20.30 24.59,20.05 26.04,19.99 27.59,19.91 28.06,19.90 32.00,19.90 Z M 32.00,17.24 C 27.99,17.24 27.49,17.26 25.91,17.33 24.34,17.40 23.27,17.65 22.33,18.01 21.36,18.39 20.54,18.90 19.72,19.72 18.90,20.54 18.39,21.37 18.01,22.33 17.65,23.27 17.40,24.34 17.33,25.92 17.26,27.49 17.24,27.99 17.24,32.00 17.24,36.01 17.26,36.51 17.33,38.09 17.40,39.66 17.65,40.73 18.01,41.67 18.39,42.65 18.90,43.47 19.72,44.29 20.54,45.11 21.37,45.61 22.33,45.99 23.27,46.36 24.34,46.61 25.92,46.68 27.49,46.75 27.99,46.77 32.01,46.77 36.02,46.77 36.52,46.75 38.09,46.68 39.66,46.61 40.74,46.36 41.68,45.99 42.65,45.62 43.47,45.11 44.29,44.29 45.11,43.47 45.62,42.64 46.00,41.67 46.36,40.74 46.61,39.66 46.68,38.09 46.75,36.51 46.77,36.01 46.77,32.00 46.77,27.99 46.75,27.49 46.68,25.91 46.61,24.34 46.36,23.27 46.00,22.33 45.62,21.35 45.11,20.53 44.29,19.71 43.47,18.89 42.65,18.39 41.68,18.01 40.74,17.64 39.67,17.39 38.09,17.32 36.51,17.26 36.01,17.24 32.00,17.24 Z"
                    />
                  </g>
                  <g
                    className="social-svg-mask"
                    style={
                      Object {
                        "MozTransition": "fill 170ms ease-in-out",
                        "OTransition": "fill 170ms ease-in-out",
                        "WebkitTransition": "fill 170ms ease-in-out",
                        "fill": "#056676",
                        "msTransition": "fill 170ms ease-in-out",
                        "transition": "fill 170ms ease-in-out",
                      }
                    }
                  >
                    <path
                      d="M0,0v64h64V0H0z M 42.03,23.99 C 42.03,24.99 41.22,25.79 40.23,25.79 39.23,25.79 38.43,24.99 38.43,23.99 38.43,22.99 39.23,22.19 40.23,22.19 41.22,22.19 42.03,22.99 42.03,23.99 Z M 24.52,31.99 C 24.52,27.74 27.97,24.29 32.22,24.29 36.47,24.29 39.92,27.75 39.92,31.99 39.92,36.24 36.47,39.70 32.22,39.70 27.97,39.70 24.52,36.25 24.52,31.99 Z M 27.22,31.99 C 27.22,34.76 29.46,36.99 32.22,36.99 34.98,36.99 37.22,34.76 37.22,31.99 37.22,29.23 34.98,27.00 32.22,27.00 29.46,27.00 27.22,29.23 27.22,31.99 Z M 38.28,19.79 C 36.70,19.72 36.22,19.70 32.22,19.70 28.22,19.70 27.74,19.71 26.17,19.79 24.69,19.85 23.90,20.11 23.37,20.31 22.67,20.58 22.17,20.90 21.65,21.43 21.13,21.96 20.80,22.46 20.53,23.16 20.33,23.68 20.08,24.48 20.01,25.94 19.94,27.52 19.92,27.99 19.92,32.01 19.92,36.01 19.94,36.48 20.01,38.06 20.08,39.52 20.33,40.32 20.53,40.84 20.80,41.54 21.13,42.05 21.65,42.57 22.17,43.10 22.67,43.41 23.37,43.69 23.90,43.90 24.69,44.15 26.17,44.21 27.74,44.28 28.22,44.30 32.22,44.30 36.22,44.30 36.70,44.28 38.28,44.21 39.75,44.15 40.54,43.89 41.07,43.69 41.77,43.42 42.27,43.10 42.80,42.57 43.32,42.05 43.64,41.54 43.91,40.84 44.12,40.32 44.36,39.52 44.43,38.06 44.50,36.48 44.52,36.01 44.52,32.01 44.52,27.99 44.50,27.52 44.43,25.94 44.36,24.48 44.12,23.68 43.91,23.16 43.64,22.46 43.32,21.96 42.80,21.43 42.27,20.90 41.77,20.59 41.07,20.31 40.54,20.10 39.75,19.85 38.28,19.79 Z M 26.03,17.09 C 27.64,17.02 28.15,17.00 32.22,17.00 36.27,17.00 36.80,17.02 38.38,17.08 38.38,17.08 38.41,17.08 38.41,17.08 40.01,17.15 41.10,17.41 42.06,17.78 43.04,18.17 43.87,18.68 44.71,19.51 45.54,20.34 46.06,21.18 46.44,22.17 46.81,23.13 47.06,24.21 47.14,25.81 47.21,27.41 47.23,27.92 47.23,31.99 47.23,36.07 47.21,36.58 47.14,38.18 47.06,39.78 46.81,40.87 46.44,41.82 46.06,42.80 45.54,43.65 44.71,44.48 43.87,45.31 43.04,45.83 42.06,46.21 41.10,46.58 40.00,46.84 38.41,46.91 36.81,46.98 36.31,47.00 32.23,47.00 28.15,47.00 27.64,46.98 26.04,46.91 24.44,46.84 23.35,46.58 22.40,46.21 21.42,45.82 20.58,45.31 19.75,44.48 18.91,43.65 18.39,42.81 18.01,41.82 17.64,40.86 17.39,39.78 17.32,38.18 17.25,36.58 17.23,36.07 17.23,31.99 17.23,27.92 17.25,27.41 17.32,25.82 17.39,24.21 17.64,23.13 18.01,22.17 18.39,21.20 18.91,20.35 19.75,19.52 20.58,18.69 21.41,18.17 22.40,17.78 23.35,17.42 24.44,17.16 26.03,17.09 Z"
                    />
                  </g>
                </svg>
              </div>
            </a>
          </li>
          <li
            className="pl-1"
          >
            <a
              aria-label="facebook"
              className="social-icon"
              href="https://www.facebook.com"
              style={
                Object {
                  "display": "inline-block",
                  "height": "50px",
                  "overflow": "hidden",
                  "position": "relative",
                  "verticalAlign": "middle",
                  "width": "50px",
                }
              }
            >
              <div
                className="social-container"
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "position": "absolute",
                    "top": 0,
                    "width": "100%",
                  }
                }
              >
                <svg
                  className="social-svg"
                  style={
                    Object {
                      "borderRadius": "50%",
                      "fillRule": "evenodd",
                      "height": "100%",
                      "left": 0,
                      "position": "absolute",
                      "top": 0,
                      "width": "100%",
                    }
                  }
                  viewBox="0 0 64 64"
                >
                  <g
                    className="social-svg-background"
                    style={
                      Object {
                        "MozTransition": "fill 170ms ease-in-out",
                        "OTransition": "fill 170ms ease-in-out",
                        "WebkitTransition": "fill 170ms ease-in-out",
                        "fill": "transparent",
                        "msTransition": "fill 170ms ease-in-out",
                        "transition": "fill 170ms ease-in-out",
                      }
                    }
                  >
                    <circle
                      cx="32"
                      cy="32"
                      r="31"
                    />
                  </g>
                  <g
                    className="social-svg-icon"
                    style={
                      Object {
                        "MozTransition": "fill 170ms ease-in-out",
                        "OTransition": "fill 170ms ease-in-out",
                        "WebkitTransition": "fill 170ms ease-in-out",
                        "fill": "transparent",
                        "msTransition": "fill 170ms ease-in-out",
                        "transition": "fill 170ms ease-in-out",
                      }
                    }
                  >
                    <path
                      d="M34.1,47V33.3h4.6l0.7-5.3h-5.3v-3.4c0-1.5,0.4-2.6,2.6-2.6l2.8,0v-4.8c-0.5-0.1-2.2-0.2-4.1-0.2 c-4.1,0-6.9,2.5-6.9,7V28H24v5.3h4.6V47H34.1z"
                    />
                  </g>
                  <g
                    className="social-svg-mask"
                    style={
                      Object {
                        "MozTransition": "fill 170ms ease-in-out",
                        "OTransition": "fill 170ms ease-in-out",
                        "WebkitTransition": "fill 170ms ease-in-out",
                        "fill": "#056676",
                        "msTransition": "fill 170ms ease-in-out",
                        "transition": "fill 170ms ease-in-out",
                      }
                    }
                  >
                    <path
                      d="M0,0v64h64V0H0z M39.6,22l-2.8,0c-2.2,0-2.6,1.1-2.6,2.6V28h5.3l-0.7,5.3h-4.6V47h-5.5V33.3H24V28h4.6V24 c0-4.6,2.8-7,6.9-7c2,0,3.6,0.1,4.1,0.2V22z"
                    />
                  </g>
                </svg>
              </div>
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</footer>
`;
